python/private/pypi/whl_library_targets.bzl

```diff
@@ -274,12 +274,18 @@ def whl_library_targets(
     if group_name and "//:" in dep_template:
         # This is the legacy behaviour where the group library is outside the hub repo
         label_tmpl = dep_template.format(
-            name = "_groups",
+            name = "_config",
             target = normalize_name(group_name) + "_{}",
+        ).replace(
+            "//:",
+            "//_groups:",
```

Comment:
Here it is a little more hacky than I would like, but this would disappear together with WORKSPACE, so it is fine for the time being - no need to add another parameter to the macro.

Reply:
SGTM. Just put a comment giving context